lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -46,6 +46,12 @@ encapsulation_dot1q:
   config_set_append: "%s encapsulation dot1q %s"
   default_value: ""
 
+fabric_frwd_anycast:
```

Comment:
For consistency with other fabric-forwarding properties this should be called `fabric_forwarding_(something)`.

Are there other fabric forwarding modes or potential for more modes in the future? If so then this should be called `fabric_forwarding_mode_(something)`, ie. `fabric_forwarding_mode_anycast_gateway` is fine.